charts/opensearch/values.yaml

```diff
@@ -404,12 +404,16 @@ networkPolicy:
 # please use the above podSecurityContext.fsGroup instead
 fsGroup: ""
 
-## Set optimal sysctl's. This requires privilege. Can be disabled if
+## Set optimal sysctl's through securityContext. This requires privilege. Can be disabled if
 ## the system has already been preconfigured. (Ex: https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html)
 ## Also see: https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/
 sysctl:
   enabled: false
 
+## Set optimal sysctl's through privileged initContainer.
```

Comment:
Good to add these options in the CI as well so that it is tested everytime : https://github.com/opensearch-project/helm-charts/tree/main/charts/opensearch/ci

Reply:
I updated all the values files :)